tests/sentry/integrations/slack/test_integration.py

```diff
@@ -141,3 +143,109 @@ def test_reassign_user(self):
         self.assert_setup_flow(authorizing_user_id='UXXXXXXX2')
         identity = Identity.objects.get()
         assert identity.external_id == 'UXXXXXXX2'
+
+
+class SlackIntegrationBotAppTest(IntegrationTestCase):
```

Comment:
This code is almost identical to `SlackIntegrationTest`, can this be abstracted?

Reply:
Yeah I'll pull out a base class